packages/server-core/src/user/avatar/avatar.hooks.ts

```diff
@@ -192,7 +192,9 @@ const updateUserAvatars = async (context: HookContext<AvatarService>) => {
     query: {
       id: {
         $ne: context.id?.toString() as AvatarID
-      }
+      },
+      isPublic: true,
+      $limit: 1000
     },
     paginate: false
```

Comment:
@aditya-mitra as we are specifiying limit in the query, then we shouldnt pass paginate:false. Also, the response should be considered as Paginated object instead of array.